src/traversal/printRelayOSSQuery.js

```diff
@@ -121,10 +121,8 @@ function printRoot(
     oneIndent + fieldName + children + newLine + '}';
 }
 
-function printMutation(
-  node: RelayQuery.Mutation,
-  printerState: PrinterState
-): string {
+function printOperation(node: RelayQuery.Operation, printerState: PrinterState): string {
+  const operationKind = node.getConcreteQueryNode().kind.toLowerCase();
```

Comment:
Yeah, maybe add `operationKind` as an argument and call the `printOperation` explicitly for `Mutation` and `Subscription` (instead of calling it for every `Operation`)?

`printOperation('subscription', node, state)`?